src/api/src/lib.rs

```diff
@@ -50,6 +50,13 @@ impl Cli {
                     .required(false)
                     .takes_value(true)
                     .help("Block device configuration. \n\tFormat: \"path=<string>\"")
+            )
+            .arg(
+                Arg::with_name("console")
+                    .long("console")
+                    .required(false)
+                    .takes_value(true)
+                    .help("Console configuration. \n\tFormat: \"type=<string>\"")
```

Comment:
What are the possible types here? For the other ones it is self explanatory what the value of the configuration is (i.e. a path, kernel_load_addr, etc), but for this one it is not. We should write the possible values in the help here, and also when we are parsing it, we should check that the value is one of the supported ones.